README.md

```diff
@@ -115,14 +104,31 @@ response.fail(String message)
 response.say("OK").send()
 ```
 
+## session
+```javascript
+// check if you can use session (read or write)
+Boolean session.isAvailable()
```

Comment:
It seems we are tolking about different. `request.session()` and `response.session()` are in use. If we change them (to object, for example), then current apps using alexa-app might be broken.